javascript-timers/interval.js

```diff
@@ -0,0 +1,16 @@
+var $countdownDisplay = document.querySelector('.countdown-display');
+var intervalID = setInterval(setCountdownInterval, 1000);
+
+function setCountdownInterval() {
+  if ($countdownDisplay.textContent === '4') {
+    $countdownDisplay.textContent = '3';
+  } else if ($countdownDisplay.textContent === '3') {
+    $countdownDisplay.textContent = '2';
+  } else if ($countdownDisplay.textContent === '2') {
+    $countdownDisplay.textContent = '1';
+  } else if ($countdownDisplay.textContent === '1') {
```

Comment:
What if the timer started from 1000? You should try to find a way to do this that doesn't require a conditional for each number.